alpine/3.10/7.2/Dockerfile

```diff
@@ -102,6 +102,13 @@ RUN version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") \
     && curl -A "Docker" -o /tmp/blackfire.so -D - -L -s https://packages.blackfire.io/binaries/blackfire-php/1.31.0/blackfire-php-alpine_amd64-php-$version.so \
     && mv /tmp/blackfire.so $(php -r "echo ini_get ('extension_dir');")/blackfire.so
 
+# Tideways
+ENV TIDEWAYS_APIKEY=''
+ENV TIDEWAYS_DAEMON_URI=''
+RUN wget https://s3-eu-west-1.amazonaws.com/tideways/extension/5.1.16/tideways-php-5.1.16-alpine.tar.gz \
+    && tar xvzf tideways-php-5.1.16-alpine.tar.gz \
+    && cp ./build/dist/tideways-php-alpine-7.2.so /usr/local/lib/php/extensions/no-debug-non-zts-20170718/tideways.so
```

Comment:
@zyuzka Can we use `$(php-config --extension-dir)/tideways.so` instead of hardcoding the folder in Dockerfile?